compliance_checker/acdd.py

```diff
@@ -400,30 +400,48 @@ def _check_total_z_extents(self, ds, z_variable):
         :param netCDF4.Dataset ds: An open netCDF dataset
         :param str z_variable: Name of the variable representing the Z-Axis
         '''
-        vert_min = float(ds.geospatial_vertical_min)
-        vert_max = float(ds.geospatial_vertical_max)
         msgs = []
         total = 2
+        try:
```

Comment:
Definitely an omission by me to not wrap these in a try-except like I had done for the lat and lon extents, which is essentially the same check. Good catch.